packages/react-components/src/components/styles/ParticipantItem.styles.ts

```diff
@@ -13,7 +15,12 @@ export const participantItemContainerStyle = mergeStyles({
   maxWidth: '20rem',
   minWidth: '12rem',
   cursor: 'pointer',
-  alignItems: 'center'
+  alignItems: 'center',
+  // Mobile height for participant items increased from default height
+  '@media screen and (max-width: 640px)': {
```

Comment:
Components don't make these decisions internally. Instead, they expose a prop that can be used to indicate that we want larger touch targets. The composite decides when to set that prop. This ensures that (a) we don't have the `@media` queries spread out over the code-base, and composites can set this consistently, (b) Contoso when using the components directly can make these decision themselves.

See `ChatMessageActionFlyout` for an example.